r55/src/exec.rs

```diff
@@ -132,7 +133,11 @@ fn execute_riscv(
     let emu = &mut rvemu.emu;
     let returned_data_destiny = &mut rvemu.returned_data_destiny;
     if let Some(destiny) = std::mem::take(returned_data_destiny) {
-        let data = emu.cpu.bus.get_dram_slice(destiny).unwrap();
+        let data = emu
+            .cpu
+            .bus
+            .get_dram_slice(destiny)
+            .unwrap_or_else(|e| panic!("Unable to get destiny dram slice ({e:?})"));
```

Comment:
Is this panic acceptable here? Probably should return Result and handle into a revert error?

Reply:
Since it was just unwrap, yes.

I’m with result too (like at r55 draft PR). Maybe we should avoid all/max unwrap and panic inside the whole crate?

Reply:
Like 1 PR for it

Reply:
yea agree